h/schemas/annotation.py

```diff
@@ -327,6 +327,12 @@ class SearchParamsSchema(colander.Schema):
         missing="updated",
         description="The field by which annotations should be sorted.",
     )
+    search_after = colander.SchemaNode(
+        colander.String(),
+        missing=colander.drop,
+        description="""Returns results after the annotation who's sort field
+                    has this value. This is used for pagination.""",
+    )
```

Comment:
I'm not sure how complicated we want to get/can get here. Technically the type of search_after depends on sort. There are some benefits to handling it at this level. For example if the user specifies an unrecognized date format it will notify them instead of just silently ignoring it.